remarkable_mouse/evdev.py

```diff
@@ -25,14 +25,22 @@ def create_local_device():
     import libevdev
     device = libevdev.Device()
 
-    # Set device properties to emulate those of Wacom tablets
-    device.name = 'reMarkable tablet'
+    # Set device properties to emulate those of Wacom tablet.
+    # "pen" or "wacom" must be in the name or gdk will treat us as a TOUCHSCREEN device
+    # rather than a PEN device.
+    #
+    # Then "Mutter", which is responsible for doing the input mapping will silently
+    # refuse to map our device when requested in gnome-control-center
+    #
+    # (see https://gitlab.gnome.org/GNOME/gtk/blob/master/gdk/x11/gdkdevicemanager-xi2.c)
+    # (See https://gitlab.gnome.org/GNOME/mutter/-/blob/master/src/backends/meta-input-settings.c)
+    device.name = 'reMarkable pen' # keep "pen" !
```

Comment:
As the device name is now `reMarkable pen`, you need to reflect that in the `xinput` assignments (eg. on 138), otherwise you'll get `device 'reMarkable tablet stylus' not found" errors.

Hint: `s/tablet/pen/`